polls/views.py

```diff
@@ -2,13 +2,12 @@
 from django.http import HttpResponseRedirect
 from django.core.urlresolvers import reverse
 from django.views import generic
-from django.utils.translation import ugettext_lazy as _
+# from django.utils.translation import ugettext_lazy as _
```

Comment:
Should be maybe just remove this line? We'll still have it around somewhere in our commit history.